tracee-rules/signatures/rego/regosig/aio.rego

```diff
@@ -0,0 +1,40 @@
+package tracee_aio
+
+__rego_metadoc__ := {
+    "id": "TRC-AIO",
+    "version": "0.1.0",
+    "name": "All In One Rego Rule",
+    "description": "This rule indexes all loaded Rego rules via one."
+}
+
+__rego_metadoc_all__[id] = resp {
+	some i
+		resp := data.tracee[i].__rego_metadoc__
+		id := resp.id
+}
+
+tracee_selected_events[eventSelector] {
```

Comment:
I think we should use `tracee_selected_events_all`. Otherwise we don't have any filtering after all.